### PR TITLE
customize application from single file

### LINKED ISSUE #45: Allow forks to customize application from single file
When users fork UniAuth to implement the system for their organization, currently they need to edit the names of the application in numerous files which is a tedious and boring process. 

To fix the above problem, we can load all such details (application name, description, etc) from a particular file (configs are best for this, checkout the config folder and how the `config.get` method works in the codebase). 

These can be then passed into the `views` from the `controllers` to display at various places.

### PR DESCRIPTION
**Pull request by kdsinghcoder**
# Issue Related to PR
- fixes #45 

# Checklist
- [x] Tests
- [x] Resolve Conflicts

# Screenshots
 ### After changing config file 
image
image
image

### CODE REVIEW

File: src/account/account.controller.ts [hunk 1]
```diff
@@ -54,7 +55,7 @@ export class AccountController {
     const { client_id } = incomingAuthDto;
     try {
       const applicationDetails = await this.accountService.validateAccessRequest(incomingAuthDto);
-      return res.render('account/o/login', { app: applicationDetails });
+      return res.render('account/o/login', { app: applicationDetails, Cname: appData.Name });
```
Comment (YashKumarVerma) [inline src/account/account.controller.ts:58]: what is cname?
can we use some easy to understand name here ?

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: src/account/account.controller.ts (hunk 1 above)
Comment (kdsinghcoder) [reply src/account/account.controller.ts:58]: company name.
If its fine.

Comment (YashKumarVerma) [reply src/account/account.controller.ts:58]: no, please change.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by kdsinghcoder — COMMENTED** [review]
Cname changes

**Pull request closed without merging**